README.md

```diff
@@ -117,3 +159,55 @@ See [LICENSE](https://github.com/monte-carlo-data/apollo-agent/blob/main/LICENSE
 ## Security
 
 See [SECURITY](https://github.com/monte-carlo-data/apollo-agent/blob/main/SECURITY.md) for more information.
+
+## Advanced deployment
```

Comment:
Let's include a comment making it clear that the preferred way to deploy is using our templates and that these templates also support using any image, so devs can just use a test image with them. 

Using IaC is easier and more manageable than creating resources manually.

Reply:
The examples in the template repos might also be handy to have links for too as they automate usage

E.g. with AWS Terraform -- https://github.com/monte-carlo-data/terraform-aws-mcd-agent/blob/70c571c5bf0e698f3b640fef8ffd03cc3e8372e1/examples/agent/Makefile#L10